### PR TITLE
Boreholes to geoSciML

### PR DESCRIPTION
**Pull request by annat2022**
SVC-1651

### CODE REVIEW

File: swisstopo-geoSciML/instances/subset_2024-02-12.gml
Comment (JohannaOtt) [inline swisstopo-geoSciML/instances/subset_2024-02-12.gml:1]: Same as for the other PR, might need to be checked again if the data should be added here.

**Review by JohannaOtt — APPROVED** [review]
LGTM

File: swisstopo-geoSciML/instances/subset_2024-02-12.gml
Comment (annat2022) [reply swisstopo-geoSciML/instances/subset_2024-02-12.gml:1]: Same here, it is agreed with swisstopo that we add some transformed sample data to the repository. I could delete the old ones and keep only the latest version to save space.

Comment (JohannaOtt) [reply swisstopo-geoSciML/instances/subset_2024-02-12.gml:1]: I think deleting the old ones won't make a difference as they are still in the repository's history and therefore also contained when cloning it. But as the fiels are really small and as it is agreed this way, I would be fine with leaving it as is.

**Pull request merged**